stories/components/BoxVaultAPY.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { BoxVaultAPY as BoxVaultAPYComponent } from "../../src";
+import styled from "styled-components";
+
+export default {
+  title: "Components/Box Vault APY",
+  component: BoxVaultAPYComponent,
+} as ComponentMeta<typeof BoxVaultAPYComponent>;
+
+interface ColsProps {
+  cols: number;
+}
+const StyledWrapper = styled.div<ColsProps>`
+  display: grid;
+  grid-template-columns: ${({ cols }) => `repeat(${cols}, minmax(0, 1fr))`};
+  gap: 1rem;
+`;
+
+const Template: ComponentStory<typeof BoxVaultAPYComponent> = (props) => (
```

Comment:
In smaller screens this looks a bit off, are we going to support responsiveness?

<img width="959" alt="Screen Shot 2022-03-08 at 9 14 47 am" src="https://user-images.githubusercontent.com/78794805/157126910-6a13a8ec-b1b7-4d10-9cab-78f568780b66.png">

Reply:
Fixed by be4c50e